base/pom.xml

```diff
@@ -88,6 +88,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.spotify</groupId>
+                <artifactId>dockerfile-maven-plugin</artifactId>
+                <configuration>
+                    <buildArgs>
+                        <!-- Redhat UBI Minimal Version -->
+                        <UBI_MINIMAL_VERSION>8.3-298</UBI_MINIMAL_VERSION>
```

Comment:
we should create maven properties for these versions and reference them here if you want to go this route. you can then override versions when building or use scripts to update the values if needed in the future.

Reply:
In the project pom and not in say common? Makes sense.